modules/admin_manual/pages/configuration/files/file_versioning.adoc

```diff
@@ -144,19 +144,19 @@ image::configuration/files/files_versions/version-metadata.png[Extended Version
 
 You can enable the feature xref:configuration/server/config_sample_php_parameters.adoc#save-additional-metadata-for-versions[via an entry in config.php]. 
 
-WARNING: The metadata is only collected after enabling the feature using xref:configuration/server/config_sample_php_parameters.adoc#save-additional-metadata-for-versions[config.php] configuration file. This means the author or version number will show only for new versions of the file and be empty for older versions.
+WARNING: The metadata is only collected after enabling the feature using the xref:configuration/server/config_sample_php_parameters.adoc#save-additional-metadata-for-versions[config.php] configuration file. This means the author or version number will show only for new versions of the file and be empty for older versions.
 
 === Show the Author of a Version
 
-Each update to the file that generates new file version will additionally show the author in the Versions Tab. This is helpful to keep track of who changed a version. 
+Each update to the file that generates a new file version will additionally show the author in the Versions Tab. This is helpful to keep track of who changed a version.
 
 === Conscious Persistent Major Versions
 
 ownCloud can automatically generate and increment version numbers for new file versions. Based on this feature, one can define the actual version to be a major version which gets a major version number and all new upcoming versions start counting from this major number. Major versions are persistent versions that are not subject to any retention policies set by ownCloud.
 
-TIP: Publishing the version persists it until the original file is deleted by somebody with such permission or by the system administrator via occ versions:cleanup command. For more information check xref:configuration/server/files/file_versioning.adoc#how-versions-are-deleted[versions retention] section.
+TIP: Publishing the version persists it until the original file is deleted by somebody with such permission or by the system administrator via occ versions:cleanup command. For more information check the xref:configuration/server/files/file_versioning.adoc#how-versions-are-deleted[versions retention] section.
```

Comment:
```suggestion
TIP: Publishing the version persists it until the original file is deleted by somebody with such permission or by the system administrator via occ versions:cleanup command. For more information check the xref:configuration/files/file_versioning.adoc#how-versions-are-deleted[versions retention] section.
```
Resolves to: `https://doc.owncloud.com/server/10.12/admin_manual/configuration/files/file_versioning.html#how-versions-are-deleted`